models/_section.json

```diff
@@ -0,0 +1,148 @@
+        }
+      },
+      "filter": "section",
+      "model": "section"
+    }
+  ],
+  "models": [
+    {
+      "id": "section",
+      "fields": [
+        {
+          "component": "multiselect",
+          "name": "style",
+          "label": "Style",
+          "options": [
+            {
+              "name": "Highlight",
+              "value": "highlight"
+            }
+          ]
+        },
+        {
+          "component": "select",
+          "name": "padding",
+          "label": "Padding",
+          "options": [
+            {
+              "name": "Default",
+              "value": ""
+            },
+            {
+              "name": "Small",
+              "value": "small"
+            },
+            {
+              "name": "Medium",
+              "value": "medium"
+            },
+            {
+              "name": "Large",
+              "value": "large"
+            },
+            {
+              "name": "Big",
+              "value": "big"
+            },
+            {
+              "name": "XBig",
+              "value": "xbig"
+            },
+            {
+              "name": "XXBig",
+              "value": "xxbig"
+            },
+            {
+              "name": "Huge",
+              "value": "huge"
+            },
+            {
+              "name": "XHuge",
+              "value": "xhuge"
+            },
+            {
+              "name": "XXHuge",
+              "value": "xxhuge"
+            }
+          ]
+        },
+        {
+          "component": "select",
+          "name": "margin",
+          "label": "Margin",
+          "options": [
+            {
+              "name": "Default",
+              "value": ""
+            },
+            {
+              "name": "Small",
+              "value": "small"
+            },
+            {
+              "name": "Medium",
+              "value": "medium"
+            },
+            {
+              "name": "Large",
+              "value": "large"
+            },
+            {
+              "name": "Big",
+              "value": "big"
+            },
+            {
+              "name": "XBig",
+              "value": "xbig"
+            },
+            {
+              "name": "XXBig",
+              "value": "xxbig"
+            },
+            {
+              "name": "Huge",
+              "value": "huge"
+            },
+            {
+              "name": "XHuge",
+              "value": "xhuge"
+            },
+            {
+              "name": "XXHuge",
+              "value": "xxhuge"
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "filters": [
+    {
+      "id": "section",
+      "components": [
+        "hero",
+        "accordion",
+        "cards",
+        "carousel",
+        "columns",
+        "fragment",
+        "quote",
+        "tabs",
+        "video",
+        "image",
+        "text",
+        "targeted-block",
+        "product-recommendations",
+        "product-teaser"
```

Comment:
This must match the block/components you actually have. For example you don't have `quote` etc.

Reply:
Corrected.

Custom `decorateSection` is imported from `commerce.js`